db/migrate/20170116200958_self_referential_orchestration_template.rb

```diff
@@ -0,0 +1,8 @@
+class SelfReferentialOrchestrationTemplate < ActiveRecord::Migration[5.0]
+  def change
+    change_table :orchestration_templates do |t|
+      t.references :parent, index: true, comment: "Nested template reference"
+      t.string :url, comment: "URL for nested/child template"
```

Comment:
Whoa...I've not yet seen comments on columns before...really interesting!  cc @chessbyte @carbonin